p2p/discovery/table.go

```diff
@@ -26,7 +25,7 @@ type Table struct {
 	selfNode *Node //info of local node
 }
 
-func NewTable(id NodeID, addr *net.UDPAddr) *Table {
+func newTable(id NodeID, addr *net.UDPAddr) *Table {
```

Comment:
so, you rename it to newTable, it's a private func?

Reply:
yes, we need keep it private